charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -6,7 +6,10 @@ global:
   scrape_timeout: 55s
 
   external_labels:
-    region: k-{{ .Values.global.region }}
+    region: {{ required ".Values.global.region missing" .Values.global.region }}
+    cluster: {{ required ".Values.global.cluster missing" .Values.global.cluster }}
+    cluster_type: {{ required ".Values.global.cluster_type missing" .Values.global.cluster_type }}
```

Comment:
What's a `cluster-type`? Also not used so far.

Reply:
So the idea is that `cluster_type` contains something like `controlplane` or `kubernikus`. This is mainly useful to simply the alert routing allow alerts from different types to be routed differently. Without this we always need to enumerate all regions in the alert routes which is verbose and difficult to maintain.